### PR TITLE
adding some menu work-wip

### PR DESCRIPTION
**Pull request by rebmullin**
hey @scottnath - definitely a WIP... but please review what i've done so far and give feedback : )  Please let me know if any questions..

note: the `menu.html` is what is created after running `node ./index.js`

thanks

### CODE REVIEW

**Comment by scottnath** [comment]
@rebmullin can you add a .gitignore and put node_modules in it please

**Comment by scottnath** [comment]
and remove all the node modules from your repo

**Comment by rebmullin** [comment]
hey @scottnath  - updated `.gitignore` and removed node_modules...

File: menu/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var fs = require('fs-extra');
+var twigger = require('./lib/twig-compiler');
```
Comment (scottnath) [inline menu/index.js]: This should point to pattern-importer's version instead

`var twigger = require('pattern-importer').twigCompiler();`

and you can now install pattern-importer from npm:

`npm install --save pattern-importer`

File: menu/lib/twig-compiler.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+'use strict';
```
Comment (scottnath) [inline menu/lib/twig-compiler.js]: this file is not needed. should use the pattern-importer version directly from pattern-imnporter

File: menu/package.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "fs-extra": "^0.18.2",
```
Comment (scottnath) [inline menu/package.json:12]: When you run `npm install --save pattern-importer`, pattern-importer should automatically show up in this list.

Comment (rebmullin) [inline menu/package.json:12]: yep - i see this happening now..

**Comment by rebmullin** [comment]
hey @scottnath - made additional updates per your notes. please review again. thanks

**Pull request merged**